.gitignore

```diff
@@ -16,3 +16,7 @@ Gemfile.lock
 
 # VS Code ignores
 .vscode/*
+
+# Helm chart dependencies
+deploy/helm/sumologic/charts
```

Comment:
I don't even see these in our code base. Why do we need to exclude explicitly?

Reply:
they will be generated if you are developing (making changes to) our helm chart to test locally. We want to be sure to not check those in.

Reply:
Got it. Thanks.